packages/eslint-plugin-template/src/rules/i18n.ts

```diff
@@ -248,15 +248,9 @@ export default createESLintRule<Options, MessageIds>({
                 )),
           )
         ) {
-          // If at least one child is a text node then we probably need i18n
```

Comment:
As the comment says: "[...] we **_probably_** need i18n". I decided to remove this as we can't guarantee that this fix is safe.